metadata.rb

```diff
@@ -4,15 +4,20 @@
 license          'Apache-2.0'
 description      'Installs/Configures mcrouter'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.0.2'
-chef_version     '>= 12.6' if respond_to?(:chef_version)
+version          '3.0.0'
+chef_version     '>= 12.7' if respond_to?(:chef_version)
 source_url       'https://github.com/evertrue/mcrouter-cookbook'
 issues_url       'https://github.com/evertrue/mcrouter-cookbook/issues'
 
-supports 'ubuntu', '= 14.04'
+supports 'ubuntu', '= 16.04'
+supports 'centos', '= 7.2'
+supports 'centos', '= 7.3'
 
-depends 'apt'
-depends 'build-essential'
-depends 'ark', '~> 3.1'
+depends 'apt', '~> 6.1'
```

Comment:
Small nit, but can you alphabetize the depends? It makes it easier to visually scan.